tests/packagedcode/data/maven/expectations/activemq-camel-pom.xml.json

```diff
@@ -31,10 +34,25 @@
   "maven_component_group_id": [
     "org.apache.activemq"
   ], 
+  "maven_contributor_email": [], 
   "maven_distribution_management_group_id": [], 
   "maven_scm_developer_connection": [], 
   "maven_component_extend_1": [], 
   "maven_developer_name": [], 
+  "maven_contributor_organization": [], 
+  "maven_dependency": [
```

Comment:
I think the approach we have with XPATH in the current maven code is flawed (not fault from you but from me). We are missing the actual groupid, the scope, optional, etc for each dep. This needs to be a proper data structure and not a plain aligned lists....